python/ray/tune/trial_executor.py

```diff
@@ -12,6 +15,18 @@
 logger = logging.getLogger(__name__)
 
 
+# Accessing environment variable could be slow.
```

Comment:
wait really? do you have some reference for this?

Reply:
I was under the impression that accessing environment variable incurs penalty for some scripting languages. But looking closely at os.py, it seems just a normal wrapped dictionary in process. So maybe not so much in this case.